get-started/08_using_compose.md

```diff
@@ -315,16 +310,13 @@ Now that we have our `docker-compose.yml` file, we can start it up!
     view the logs for a specific service, you can add the service name to the end of the logs command (for example,
     `docker-compose logs -f app`).
 
->**Pro tip**  Waiting for the DB before starting the app
```

Comment:
I think we can keep the "pro tip"

Reply:
Believe this is the only section where we have Pro tips. I'd be inclined to use "Note" instead for consistency.

Reply:
We have `Tip:` in a couple of places (e.g. https://docs.docker.com/config/containers/logging/configure/)

I think a "note" is more to highlight / go further into details of a change/example, whereas a "tip" is slightly more "consider this" (but not required to do, so "ok to skip" when reading)